app/views/newflow/login_signup/educator_complete_form.html.erb

```diff
@@ -135,7 +135,7 @@
                 <%= I18n.t(:"educator_complete_form.select_option") %>
               </div>
               <%= fh.select name: :books_used,
-                            options: Settings::Subjects.map {|(short_name, info)| [  info["title"], short_name] },
+                            options: @book_titles,
```

Comment:
be sure to ask @RoyEJohnson if his mult-select control is going to be redesigned to group titles by subject. that would have implications here.

Reply:
Future feature :). Gonna make that improvement in the future. For now, a simple array works. 

In the future because we only have you in the team till today 😢 .